proposals/stage-1/CHARTER_REVIEW/charter_review.md

```diff
@@ -0,0 +1,52 @@
+* If the project would like to request legal review, note that in the issue as well (this is optional at the request of the project or the CPC).
+
+### Submitting an update to an existing charter
+
+If your project has decided that its existing charter document needs to be updated, complete the following steps:
+
+* Open a pull request to update your previously approved charter. Make sure changes remain in the format of the PROJECT_CHARTER_TEMPLATE.md.
+* File an issue in the Cross-Project-Council repository - Titled *Update {$Project Name} Charter Document* and include a link to this pull request.
+* Tag '@openjs-foundation/cpc' on the issue - this notifies all CPC members that a document is ready for their review.
+* Add the 'cross-project-council-agenda' label to the issue - this will ensure the issue is taken up at the next CPC meeting. 
+* If the project would like to request legal review, note that in the issue as well (this is optional at the request of the project or the CPC).
+
+## Reviewing a charter
+
+Project Charters are the responsibility of the CPC. When reviewing a charter, the CPC shall evaluate:
+
+1. The completeness of the document - all sections marked required in the PROJECT_CHARTER_TEMPLATE.md are filled out. Optional sections that have not been filled out are marked as intentionally omitted.
+1. The comprehensibility of the document - a person familiar with the field but unfamiliar with the project should be able to read the charter and have a sound understanding of where it fits in the ecosystem. 
+1. The accuracy of the document - the document should reflect the reality of the project. It should avoid hyperbolic language (e.g. "world's best developer experience"). 
+1. The lifespan of the document - the CPC should take note of what factors may cause the charter document to change. If data that may change frequently appears in the charter - such as a list of names - the CPC may ask to remove it. 
+
+If the project handles PII, deals with a new intersection between technology and law, or would simply feel more comfortable, they or the CPC Board Representative may request review from Foundation legal counsel. This is an optional step.
+
+CPC members should add themselves as reviewers to the associated charter issue. Comments or suggested edits to the charter document should be captured on the Pull Request. 
+
+## Approving a charter
+
+Project Charters should be reviewed and approved within two weeks of submission. An exception may be made if the charter is submitted during a holiday period (such as the end of the year) or in the event either the project or CPC has requested legal review. 
+
+Charters are approved when:
+
+* Reviewing CPC members have added themselves as reviewers on the issue and approved.
+* More than 50% of voting CPC members have approved, OR 
+* More than 2/3 of all CPC members have approved, OR
+* A period of 14 days has elapsed with no concerns or objections raised. 
```

Comment:
This https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#aproving-project-charters in the governance doc says:

The request is approved when:

there are no outstanding objections
there are two or more approvals by voting CPC members
the board has been consulted in the case of substantial changes
The issue has been open for at least 14 days

I'd suggest we use that, and possibly just link to that so we don't need to maintain in 2 places